### PR TITLE
Update tooltip and themes with better visibility

### PR DESCRIPTION
**Pull request by psybers**
Reviving #2815.

### CODE REVIEW

**Comment by psybers** [comment]
@VoltaicGRiD I revived your old PR and updated VRT.

**Comment by youngcw** [comment]
The tooltips in light theme seem off.  Like it has a shadow over the whole thing.  The dark modes seem fine.

**Comment by psybers** [comment]
> https://deploy-preview-3298.demo.actualbudget.org

I think I see what you mean. The background isn't white, like the table cells, it is somewhat off.

**Comment by psybers** [comment]
image

**Comment by psybers** [comment]
@youngcw How about this?

**Comment by youngcw** [comment]
> image

thats better

**Comment by youngcw** [comment]
This adds borders around the payee and category pickers.  It would be nice if those stayed the same.

File: packages/desktop-client/e2e/schedules.test.js-snapshots/Schedules-creates-a-new-schedule-posts-the-transaction-and-later-completes-it-9-chromium-linux.png
[the remark this answers is not in the document]
Comment (psybers) [reply packages/desktop-client/e2e/schedules.test.js-snapshots/Schedules-creates-a-new-schedule-posts-the-transaction-and-later-completes-it-9-chromium-linux.png:1]: It does. Not sure why it hovers now when it did not before.

**Comment by psybers** [comment]
> This adds borders around the payee and category pickers. It would be nice if those stayed the same.

@youngcw Changed. None of the autocomplete menus show the border now.

File: packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-category-4-chromium-linux.png
Comment (youngcw) [inline packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-category-4-chromium-linux.png:1]: Can we get the styling to not change at all? This is adding extra padding or something. Same with most of the other vrt changes.

File: packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-date-4-chromium-linux.png
Comment (youngcw) [inline packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-date-4-chromium-linux.png:1]: This one is still getting a border, might be getting more padding too.

File: packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-category-4-chromium-linux.png
Comment (psybers) [reply packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-category-4-chromium-linux.png:1]: Fixed the padding.  The VRT still shows a tiny bit of something happening, not sure what exactly.

File: packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-date-4-chromium-linux.png
Comment (psybers) [reply packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-date-4-chromium-linux.png:1]: Fixed.

File: packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-category-4-chromium-linux.png
Comment (psybers) [reply packages/desktop-client/e2e/transactions.test.js-snapshots/Transactions-filters-transactions-by-category-4-chromium-linux.png:1]: @youngcw It's like it is a couple pixels narrower?  I'm not sure what is going on there.  I confirmed via inspection that the width is the same on both. It looks fine when you actually use the app.

**Comment by youngcw** [comment]
Im guessing that you are letting those filter tooltips be different on purpose.  Thats fine, but the VRT needs updated.

**Comment by psybers** [comment]
> Im guessing that you are letting those filter tooltips be different on purpose. Thats fine, but the VRT needs updated.

I'm not sure why those VRT fail. The new version has an extra filter button for tags, but the 'expected' does not.  I did not touch any filters...

File: packages/desktop-client/src/style/themes/light.ts [hunk 1]
```diff
@@ -208,3 +208,7 @@ export const budgetHeaderOtherMonth = colorPalette.gray80;
 export const floatingActionBarBackground = colorPalette.purple400;
 export const floatingActionBarBorder = floatingActionBarBackground;
 export const floatingActionBarText = colorPalette.navy50;
+
+export const tooltipText = colorPalette.navy900;
+export const tooltipBackground = colorPalette.white;
+export const tooltipBorder = colorPalette.navy150;
```
[the remark this answers is not in the document]
Comment (psybers) [reply packages/desktop-client/src/style/themes/light.ts:214]: Swing and a miss... suggesting things that were already done.

**Review by youngcw — APPROVED** [review]

**Comment by youngcw** [comment]
I don't know why those comments decided to come back a second time.

**Pull request merged**